test/content/ContentManagerTests.js

```diff
@@ -168,7 +168,7 @@ describe('Content Manager Contract Tests', () => {
             await content.connect(craftingSystemAddress).mintBatch(mintData);
             expect(await content.totalSupply(1)).to.equal(100);
 
-            await expect(content.connect(deployerAddress).mintBatch(mintData)).to.be.reverted;
+            await expect(content.connect(playerAddress).mintBatch(mintData)).to.be.reverted;
```

Comment:
This line is my mistake actually. It's a remnant from when I tested register operators "var approvalPairs4 = [[deployerAddress.address, null]];". Where null would compile the same as false. I wrote line 171 to prove that deployerAddress had been revoked the minter role. And when we decided to just leave a comment instead of writing a test for it, I forgot to also delete line 171.

Reply:
okay, I'll delete it. no problem